airbyte-cdk/python/airbyte_cdk/sources/streams/http/http.py

```diff
@@ -336,6 +336,37 @@ def _send_request(self, request: requests.PreparedRequest, request_kwargs: Mappi
             backoff_handler = default_backoff_handler(max_tries=max_tries, factor=self.retry_factor)
             return backoff_handler(user_backoff_handler)(request, request_kwargs)
 
+    def parse_response_error_message(self, response: requests.Response) -> Optional[str]:
+        # default logic to grab error from common fields
+
+        def _try_get_error(value):
+            if type(value) is str:
```

Comment:
when checking if something is a particular type in python it's better to use `isinstance` since it considers inheritance